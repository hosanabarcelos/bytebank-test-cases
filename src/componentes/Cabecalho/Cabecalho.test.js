import { render, screen } from '@testing-library/react';
import Cabecalho from './index';

test('Whether the logged in user name was rendered', () => {
    render(<Cabecalho />)

    const userName = screen.getByText('Joana Fonseca Gomes');

    expect(userName).toBeInTheDocument();
    expect(userName).toMatchSnapshot();
});

test('Renders the logo correctly', () => {
    const { getByTestId } = render(<Cabecalho />);
    const logo = getByTestId('logo');

    expect(logo).toBeInTheDocument();
});

test('Renders user avatar correctly', () => {
    const { getByAltText } = render(<Cabecalho />);
    const avatar = getByAltText('Ícone de um avatar de usuário');

    expect(avatar).toBeInTheDocument();
});