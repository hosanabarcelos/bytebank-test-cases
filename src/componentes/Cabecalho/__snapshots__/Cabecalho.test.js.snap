// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Whether the logged in user name was rendered 1`] = `
<p>
  Joana Fonseca Gomes
</p>
`;
